src/tags/object/List.js

```diff
@@ -101,47 +80,27 @@ const Model = types
 
       self._value = value;
     },
-
-    createResult(data) {
-      self.annotation.createResult({}, { ranker: data }, self, self);
-    },
-
-    updateResult(newData) {
-      // check if result exists already, since only one instance of it can exist at a time
-      if (self.result) {
-        self.result.setValue(newData);
-      } else {
-        self.createResult(newData);
-      }
-    },
-
-    // Create result on submit if it doesn't exist
-    beforeSend() {
-      if (self.result) return;
-
-      if (self.mode === 'rank') {
-        self.createResult(self._value.map(item => item.id));
-      } else {
-        self.createResult([]);
-      }
-    },
   }));
 
-const RankerModel = types.compose('RankerModel', Base, ProcessAttrsMixin, AnnotationMixin, Model);
+const ListModel = types.compose('ListModel', Base, ProcessAttrsMixin, AnnotationMixin, Model);
 
-const HtxRanker = inject('store')(
+const HtxList = inject('store')(
   observer(({ item }) => {
     const data = item.dataSource;
 
     if (!data) return null;
+    // Ranker tag will display all items in interactive mode
+    if (item.ranker) return null;
 
     return (
-      <Ranker inputData={data} handleChange={item.updateResult} />
+      <React.StrictMode>
```

Comment:
Why the inclusion of StrictMode here?

Reply:
Honestly I don't know... There are no errors in their implementation in strict mode, so yes, we can just remove it. And this strict wrapper as well. thanks!

Reply:
Are we leaving this on, or removing this?